front/app/containers/App/index.tsx

```diff
@@ -94,14 +94,9 @@ const App = ({ children }: Props) => {
   const [userSuccessfullyDeleted, setUserSuccessfullyDeleted] = useState(false);
 
   const [locale, setLocale] = useState<SupportedLocale | null>(null);
-  const [disableScroll, setDisableScroll] = useState(false);
 
   const redirectsEnabled = useFeatureFlag({ name: 'redirects' });
 
-  const franceConnectLoginEnabled = useFeatureFlag({
-    name: 'franceconnect_login',
-  });
-
```

Comment:
This was related to the full-screen mode. Since we don't seem to use that mode anymore, this code can go, too.